sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/PartitionBasedLoadBalancer.java

```diff
@@ -328,13 +328,21 @@ private void claimOwnership(final Map<String, PartitionOwnership> partitionOwner
 
         checkpointStore
             .claimOwnership(ownershipRequest)
-            .timeout(Duration.ofSeconds(1)) // TODO: configurable
+            .timeout(Duration.ofMinutes(1)) // TODO: configurable
```

Comment:
We have a set of "RetryOptions" and within there, there is a "TryTimeout", which we use as an "operation timeout". (How long should I wait for this operation to complete.) Should we reuse this same field?

Reply:
This is a timeout for checkpoint store, not for event hubs operations.